src/__tests__/config-service.test.ts

```diff
@@ -3,7 +3,7 @@ import * as path from 'node:path'
 import { createConfigService } from '../config-service'
 import { IDependencySection } from '../types'
 
-const testDirectory = path.resolve(__dirname, 'fixtures')
+const testDirectory = path.resolve(import.meta.dirname, 'fixtures')
```

Comment:
This is faux ESM. Luckily for us, Vitest transpiles this to CJS so it can use `node:vm` to run tests, but it's still weird and we might as well fix it.